lib/uplink/project.go

```diff
@@ -85,6 +82,10 @@ func (cfg *BucketConfig) setDefaults() {
 	if cfg.Volatile.RedundancyScheme.TotalShares == 0 {
 		cfg.Volatile.RedundancyScheme.TotalShares = 95
 	}
+	if cfg.EncryptionParameters.BlockSize == 0 {
+		cfg.EncryptionParameters.BlockSize = memory.KiB.Int32() *
+			int32(cfg.Volatile.RedundancyScheme.RequiredShares)
+	}
 	if cfg.Volatile.RedundancyScheme.ShareSize == 0 {
 		cfg.Volatile.RedundancyScheme.ShareSize = (1 * memory.KiB).Int32()
 	}
```

Comment:
I wonder if it should be like this, set shareSize first, then blocksize based off that: 

```
if cfg.Volatile.RedundancyScheme.ShareSize == 0 {
    cfg.Volatile.RedundancyScheme.ShareSize = (1 * memory.KiB).Int32()
}
if cfg.EncryptionParameters.BlockSize == 0 {
    cfg.EncryptionParameters.BlockSize = cfg.Volatile.RedundancyScheme.ShareSize *
        int32(cfg.Volatile.RedundancyScheme.RequiredShares)
}
```

Reply:
you're right, yep! okay, need to fix